### ISSUE TITLE
Not support new Bilibili bangumi（B站番剧） url

### ISSUE THREAD

**Issue by tzw0745** [rg3/youtube-dl#16051]
### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2018.03.26.1*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2018.03.26.1**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [x] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser

### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with youtube-dl)
- [x] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '-F', 'https://www.bilibili.com/bangumi/play/ss1359/']
[debug] Encodings: locale cp936, fs mbcs, out cp936, pref cp936
[debug] youtube-dl version 2018.03.26.1
[debug] Python version 3.5.0 (CPython) - Windows-10.0.10240
[debug] exe versions: ffmpeg N-86383-g4705edb, ffprobe N-86383-g4705edb
[debug] Proxy map: {}
[generic] ss1359: Requesting header
WARNING: Falling back on generic information extractor.
[generic] ss1359: Downloading webpage
[generic] ss1359: Extracting information
ERROR: Unsupported URL: https://www.bilibili.com/bangumi/play/ss1359/
```

### Description of your *issue*, suggested solution and other information

Bilibili change it's bangumi play url, from `https://bangumi.bilibili.com/anime/{anime_id}/play#{play id}` to `https://www.bilibili.com/bangumi/play/{play_id}/`, so youtube-dl can't handle this url now.

**Comment by tzw0745** [comment]
![image](https://user-images.githubusercontent.com/11869671/38160615-5915ede6-34f3-11e8-8853-ecf652ff79db.png)
`https://www.bilibili.com/bangumi/play/ss1359/` is playable in my chrome.

**Comment by BlackstormFirst** [comment]
hi, still not available with youtube-dl version 2018.04.16.

i.e Other URL to try (no account required):
- https://www.bilibili.com/bangumi/play/ep103088
- https://www.bilibili.com/bangumi/play/ep100536

**Comment by remitamine** [comment]
Duplicate of #15545.
